aip/0162.md

```diff
@@ -111,6 +111,44 @@ user sent a name with no revision ID, the returned `name` string **must not**
 include the revision ID either. Similarly, if the user sent a name with a
 revision ID, the returned `name` string **must** explicitly include it.
 
+### Tagging revisions
+
+APIs implementing resource revisions **may** provide a mechanism for users to
+tag a specific revision with a user provided name by implementing a `Tag`
```

Comment:
Maybe call it "TagRevision" here given that it's `:tagRevision` and `TagBookRevision`?

Reply:
We have `ListBookRevisions`, though.
Also, there could be more than one resource in the same API with revisions.

Reply:
I'm not sure you understood my suggestion. I'm not suggesting any changes to the service itself - just the description. I'd call it a `TagRevision` method instead of a `Tag` method. That's the only suggestion I'm making.

Reply:
Oh, you are right, I totally misunderstood. :-)